deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md

```diff
@@ -1,34 +1,34 @@
 ---
-mapped_pages:
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-manage-repositories.html
+navigation_title: "Elastic Cloud Enterprise"
+
+applies_to:
+  deployment:
+    ece: 
 ---
 
-# Elastic Cloud Enterprise [ece-manage-repositories]
+# Manage snapshot repositories in Elastic Cloud Enterprise [ece-manage-repositories]
 
-Snapshot repositories are managed for your entire Elastic Cloud Enterprise installation and can be specified for an Elasticsearch cluster when you create or manage it.
+Snapshots enable you to back up and restore Elasticsearch indices, protecting data from accidental deletion and supporting migration between clusters. In Elastic Cloud Enterprise (ECE), snapshot repositories are managed at the platform level and can be assigned to individual deployments.
 
-When a repository is specified, a snapshot is taken every 30 minutes by default. The interval can be adjusted on per deployment basis.
+When a repository is assigned to a deployment, a snapshot is taken every 30 minutes by default. The snapshot interval can be adjusted per deployment.
 
-Snapshots are configured and restored using the [snapshot and restore feature](../snapshot-and-restore.md).
+## Supported repository types
 
-Elastic Cloud Enterprise installations support the following {{es}} [snapshot repository types](/deploy-manage/tools/snapshot-and-restore/self-managed.md#ess-repo-types):
+Elastic Cloud Enterprise installations support the following {{es}} snapshot repository types:
 
-* [Azure](/deploy-manage/tools/snapshot-and-restore/azure-repository.md)
-* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/google-cloud-storage-repository.md)
-* [AWS S3](/deploy-manage/tools/snapshot-and-restore/s3-repository.md)
+* [Azure](/deploy-manage/tools/snapshot-and-restore/azure-storage-repository.md)
+* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/google-cloud-storage-gcs-repository.md)
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/ece-aws-custom-repository.md), including [Minio](/deploy-manage/tools/snapshot-and-restore/minio-on-premise-repository.md)
```

Comment:
Let's remove the links as they are the same we offer a bit later on `add repository` section.
And let's place AWS S3 the first one.

```suggestion
* AWS S3
* Azure Blob Storage
* Google Cloud Storage
* Minio S3
```